### DISCUSSION TITLE
"Thumbnail Previews Missing in SwarmUI"

### DISCUSSION THREAD

**Discussion by iresolvers** [Stability-AI/StableSwarmUI#278] [General]
None of the image thumbnail Previews at the bottom are working in my Stable SwarmUI I'm not sure why...
![Missing Previews](https://github.com/Stability-AI/StableSwarmUI/assets/73430577/de82fc39-e6ad-4f68-abe6-b03ee3100494)

**Comment by mcmonkey4eva** [comment]
that's not part of Comfy nor Swarm, that's a separate addon -- you'd have to ask the developer of whatever provides that. Likely it's this issue again https://github.com/comfyanonymous/ComfyUI/pull/867

**Reply by iresolvers** [reply]
O ok thanks for the input I will find out about that plugin thanks so much!
